### DISCUSSION TITLE
LOVE GH Sponsors over Patreon but lack of discord role integration is a pain point for my community

### DISCUSSION THREAD

**Discussion by tallguyjenks** [community/community#4340] [Other Feature Feedback, Questions, & Ideas]
I had a video call with **Darby Thomas** in February about this but something I rely a lot on for my community is custom discord roles based on sponsorship tier. Patreon does this beautifully, but i'd love to shift all sponsors to Github and have custom roles set based on sponsorship tier. This would mean that when the person discontinues sponsorship they lose their discord role and the privileges that role bestows them in my server. 

This is probably the only major pain point I have with GH sponsors.

<details>
    <summary>Image examples and reference</summary>
    <img src="https://user-images.githubusercontent.com/29872822/123503444-f21c7b00-d607-11eb-8f03-f941e0aa9a7f.png"/>
    <img src="https://user-images.githubusercontent.com/29872822/123503454-f8aaf280-d607-11eb-8d58-4d35bda89767.png"/>
    <img src="https://user-images.githubusercontent.com/29872822/123503474-0b252c00-d608-11eb-97c5-22ad91a4edfa.png"/>
    <img src="https://user-images.githubusercontent.com/29872822/123503520-4aec1380-d608-11eb-81fc-27c3b3a8d896.png"/>
</details>

**Comment by bdresser** [comment]
Thanks for the feedback @tallguyjenks! This is on our roadmap, so stay tuned. We may reach out in the future for some UX feedback if that's okay with you. 

For others reading the thread, we'd love to hear more about your usecase and how you'd expect the integration to work. 

Will followup here with more updates in the coming months :)

**Reply by tallguyjenks** [reply]
absolutely!

**Reply by PopWarner** [reply]
It's been a few years, curious if there has been any activity on this?

**Reply by LlmDl** [reply]
I subscribed to this a while ago and I am also curious.

**Comment by leaanthony** [comment]
Any updates? There are loads of people looking for this online....
